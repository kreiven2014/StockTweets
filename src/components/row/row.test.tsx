/* REACT */
import React from "react";
import { View } from "react-native";

/* MODULES */
import renderer from "react-test-renderer";

/* CUSTOM MODULES */
import Row from "./row";

describe("Row tests", () => {
  test("renders correctly male", (): void => {
    const tree = renderer
      .create(
        <Row>
          <View />
        </Row>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
