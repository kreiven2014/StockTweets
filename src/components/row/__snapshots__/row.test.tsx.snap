// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row tests renders correctly male 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View />
</View>
`;
